js/repl/Repl.js

```diff
@@ -222,26 +229,40 @@ class Repl extends React.Component<Props, State> {
           showOfficialExternalPlugins={state.showOfficialExternalPlugins}
           loadingExternalPlugins={state.loadingExternalPlugins}
         />
-
-        <div className={styles.panels}>
-          <CodeMirrorPanel
-            className={styles.codeMirrorPanel}
-            code={state.code}
-            errorMessage={state.compileErrorMessage}
-            fileSize={state.meta.rawSize}
-            onChange={this._updateCode}
-            options={options}
-            placeholder="Write code here"
-          />
-          <CodeMirrorPanel
-            className={styles.codeMirrorPanel}
-            code={state.compiled}
-            errorMessage={state.evalErrorMessage}
-            fileSize={state.meta.compiledSize}
-            info={state.debugEnvPreset ? state.envPresetDebugInfo : null}
-            options={options}
-            placeholder="Compiled output will be shown here"
-          />
+        <div className={styles.wrapperPanels}>
+          <div
+            className={[
```

Comment:
I think this can be slightly cleaner with emotion's `cx`:

```jsx
import { cs } from 'emotion';

<div className={cx(styles.panels, !state.timeTravel && styles.panelsMax)} />
```

Reply:
@existentialism changed to use `cx`!